src/raytracer/Raytracer.m

```diff
@@ -81,45 +100,52 @@
 switchPolarization = 0;
 switchCp = 0;
 polarizationTx = [1, 0];
+[~, scenarioName] = fileparts(paraCfgInput.inputScenarioName);
 
 MaterialLibrary = importMaterialLibrary(paraCfgInput.materialLibraryPath);
 
 % Extracting CAD file and storing in an XMl file, CADFile.xml
 [CADop, switchMaterial] = getCadOutput(paraCfgInput.environmentFileName,...
     inputPath, MaterialLibrary, paraCfgInput.referencePoint,...
     paraCfgInput.selectPlanesByDist, paraCfgInput.indoorSwitch);
-
+% staticCad = CADop;
 if paraCfgInput.switchSaveVisualizerFiles == 1
     % Save output file with room coordinates for visualization
     RoomCoordinates = CADop(:, 1:9);
     csvwrite(fullfile(visualizerPath, 'RoomCoordinates.csv'),...
         RoomCoordinates);
 end
 
-%% Loop over time instances
-for iterateTimeDivision = 1:paraCfgInput.numberOfTimeDivisions
-    if mod(iterateTimeDivision,100)==0 && displayProgress 
-        disp([fprintf('%2.2f', iterateTimeDivision/paraCfgInput.numberOfTimeDivisions*100),'%'])
+%% Node-node ray tracing
+if paraCfgInput.nodeMobility
+    T = paraCfgInput.numberOfTimeDivisions;
```

Comment:
you used T in the docstring, it might create confusion. it might also be a good idea to use a clearer variable name